### PR TITLE
Reduce the game descriptions whitespace.

### LINKED ISSUE #635: Whitespace isn't properly stripped from game description
Not sure how this happened

http://s15.postimg.org/v5gysgbwp/Screen_Shot_2015_01_28_at_8_36_28_PM.png

### PR DESCRIPTION
**Pull request by kaiserfro**
Fixes #635

### CODE REVIEW

**Comment by ZeldaZach** [comment]
This looks and works fine, relating it to the QString docs (http://qt-project.org/doc/qt-4.8/qstring.html#simplified)

Good to merge upon passage of Travis.

**Comment by Daenyth** [comment]
This isn't sufficient to fix the problem. Servatrice needs to strip it too since the client might be out of date or malicious

**Comment by kaiserfro** [comment]
@Daenyth I made a change to the server game constructor in the common code.  I have not looked at the server code before, but this seemed to do the trick.  Have a look.

I tested it using the old code in the client, sending tabs and newlines into the game description.  The patched server seemed to strip it out.  Hopefully this is done in the right place.

**Comment by Psithief** [comment]
I don't like the unnecessary whitespace changes in this commit. Where do I see the actual changes? :-1:

File: cockatrice/src/dlg_creategame.cpp [hunk 1]
```diff
@@ -138,29 +138,29 @@ DlgCreateGame::DlgCreateGame(const ServerInfo_Game &gameInfo, const QMap<int, QS
...
-    
+
     setWindowTitle(tr("Game information"));
 }
 
 void DlgCreateGame::actOK()
 {
     Command_CreateGame cmd;
-    cmd.set_description(descriptionEdit->text().toStdString());
+    cmd.set_description(descriptionEdit->text().simplified().toStdString());
```
Comment (ctrlaltca) [inline cockatrice/src/dlg_creategame.cpp]: just a note to actually find the relevant change

File: common/server_game.cpp [hunk 2]
```diff
@@ -77,9 +76,10 @@ Server_Game::Server_Game(const ServerInfo_User &_creatorInfo, int _gameId, const
 {
     currentReplay = new GameReplay;
     currentReplay->set_replay_id(room->getServer()->getDatabaseInterface()->getNextReplayId());
-    
+    description = _description.simplified();
+
```
Comment (ctrlaltca) [inline common/server_game.cpp]: another dummy note to actually find the relevant change

**Comment by Daenyth** [comment]
Add ?w=1 to the url on the diff page

**Comment by ctrlaltca** [comment]
Nice trick, thank you!

**Comment by Daenyth** [comment]
I think this is ready to merge unless anyone had further feedback

**Comment by ZeldaZach** [comment]
I'm fine with it

**Pull request merged**